client/app/lib/popover_view.coffee

```diff
@@ -28,31 +28,42 @@ module.exports = class PopoverView extends BaseView
             trigger: 'manual'
             title: @titleTemplate @getRenderData()
             html: true
-            placement: @getDirection()
+            placement: @getDirection
```

Comment:
I remove parenthesis, so the `getDirection` function is called by the popover plugin at `show` invocation, not at popover instanciation. This way, we can get the `tip` element (the popover content) as first arg in the function, and use it to compute dimensions dynamically.

Bootstrap 3 introduces an `auto` placement value, but unfortunately this one isn't available in Bootstrap 2.3.1.